pyproject.toml

```
@@ -1,63 +1,47 @@
-
-[build-system]
-requires = ["flit_core >=3.2,<4"]
-build-backend = "flit_core.buildapi"
-
-[project]
+[tool.poetry]
 name = "udata-hydra"
+version = "0.2.0.dev"
 description = "Async crawler and datalake service for data.gouv.fr"
-authors = [{name = "Opendata Team", email = "opendatateam@data.gouv.fr"}]
-readme = "README.md"
-license = {text = "MIT"}
-dynamic = ["version"]
-classifiers = [
-    "Development Status :: 5 - Production/Stable",
-    "Intended Audience :: Developers",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python",
-    "Programming Language :: Python :: 3.9"
-]
-dependencies = [
-    "pandas==1.3.3",
-    "cchardet==2.1.7",
-    "aiohttp==3.8.1",
-    "asyncpg==0.25.0",
-    "boto3==1.21.21",
-    "humanfriendly==10.0",
-    "marshmallow==3.14.1",
-    "minicli==0.5.0",
-    "progressist==0.1.0",
-    "python-dateutil==2.8.2",
-    "python-dotenv==0.19.2",
-    "python-magic==0.4.25 ",
-    "redis==4.1.4",
-    "requests==2.27.1",
-    "udata_event_service==0.0.8"
-]
+authors = ["Opendata Team <opendatateam@data.gouv.fr>"]
```

Should we replace by udata?
```suggestion
authors = ["Etalab <udata@data.gouv.fr>"]
```

---

I've been thinking about it, not sure. We don't we want this address (I modified it) spammed. Maybe change the name to Etalab and create a throwaway email — since apparently an email address is required by poetry?

---

As you want. We can also keep opendatateam for now and update it when we deal with https://github.com/etalab/data.gouv.fr/issues/887